public/js/recipe.js

```diff
@@ -0,0 +1,19 @@
+// Get references to page elements
+var recipeName = $("#recipeName");
+var recipeImage = $("#recipeImage");
+var prepTime = $("#prepTime");
+var servings = $("#servings");
+
+// The API object contains methods for each kind of request we'll make
+var API = {
+  getRecipe: function(recipe) {
+    return $.ajax({
+      url: "api/recipe",
```

Comment:
Just making a note here that the url will eventually need to be updated to reflect the end point.